src/Sentry.Extensions.Logging/SentryLoggingOptionsSetup.cs

```diff
@@ -15,6 +15,8 @@ public SentryLoggingOptionsSetup(ILoggerProviderConfiguration<SentryLoggerProvid
         _config = config.Configuration;
     }
 
+    [RequiresDynamicCode("Calls Microsoft.Extensions.Configuration.ConfigurationBinder.Bind(Object)")]
+    [RequiresUnreferencedCode("Calls Microsoft.Extensions.Configuration.ConfigurationBinder.Bind(Object)")]
```

Comment:
@jamescrosswell do you know by chance what's up with that?

Reply:
No idea. At first glance, the Check failure is nonsense.

Reply:
Unless perhaps it's a consequence of the other Check failures below:
> The type 'RequiresDynamicCodeAttribute' exists in both 'Sentry, Version=4.0.0.0, Culture=neutral, PublicKeyToken=fba2ec45388e2af0' and 'System.Runtime, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'

Reply:
So we added the code it's complaining about in https://github.com/getsentry/sentry-dotnet/commit/b84faa879b5e383f9c07fe887b33763d78448c2c... the whole purpose of which was to avoid Dynamic Code in the configuration binding. 

The inclusion of `Microsoft.Extensions.Configuration.Binder` and setting `<EnableConfigurationBindingGenerator>true</EnableConfigurationBindingGenerator>` in the project file mean that call to Bind will be replaced by a source generated binder... so there isnt' any dynamic code. 

If we add the two attributes above, it will break AOT Compatibility (unecessarily).